javascript/event-recommender-part2.md

```diff
@@ -0,0 +1,48 @@
+# Event Recommender System with Javascript - Part 2
+
+
+### Primary Goals
+To build UI and client-side tests for a portfolio-ready client-side application.
+
+### Overview
+You'll build an Event Recommender app that will
+help you practice building a complete client-side Object Oriented application.
+
+### Context
+Now that you've finished Part 1 and your Event Recommender class is complete, you will create a UI that allows a user to implement your class methods in a tested, deployed website.
+
+### User Requirements
+
+1. Create a User Interface with *jQuery* for *each* of the following methods:
+
+* Users:
+    * Add a new User to the System
+    * Save the Events that User wish to go
+    * Delete a User from the System
+* Event:
+    * Add a new Event to the System
+    * Update an Event's categories
+    * Delete an Event from the System
+* Filter:
+    * All Events in a certain category (order does not matter)
```

Comment:
Remove line 27, change line 28 to “You can use two separate filters or...”

Reply:
Removed this too